libraries/LittleFS/src/LittleFS.h

```diff
@@ -0,0 +1,446 @@
+/*
+ LittleFS.h - Filesystem wrapper for LittleFS on the ESP8266
+ Copyright (c) 2019 Earle F. Philhower, III.  All rights reserved.
+
+ Based heavily off of the SPIFFS equivalent code in the ESP8266 core
+ "Copyright (c) 2015 Ivan Grokhotkov. All rights reserved."
+
+ This code was influenced by NodeMCU and Sming libraries, and first version of
+ Arduino wrapper written by Hristo Gochkov.
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2.1 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library; if not, write to the Free Software
+ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+
+#ifndef __LITTLEFS_H
+#define __LITTLEFS_H
+
+#include <limits>
+#include <FS.h>
+#include <FSImpl.h>
+#include <debug.h>
+#include <flash_utils.h>
+
+#define LFS_NAME_MAX 32
+#include "../lib/littlefs/lfs.h"
+
+using namespace fs;
```

Comment:
Please put everything from this point on, including this "using fs" line, until just before the GLOBAL_INSTANCE line at the end inside a namespace. I suggest LittleFSImplementation, but it could be anything, really.
See comment at the end as well.

Reply:
Done latest commit